Algorithm.Framework/Portfolio/EqualWeightingPortfolioConstructionModel.cs

```diff
@@ -131,12 +173,17 @@ public override IEnumerable<IPortfolioTarget> CreateTargets(QCAlgorithm algorith
 
             var errorSymbols = new HashSet<Symbol>();
 
-            // Determine target percent for the given insights
-            var percents = DetermineTargetPercent(lastActiveInsights);
+            // Determine target percent for the given insights that respect the portfolio bias
+            var percents = DetermineTargetPercent(lastActiveInsights.Where(RespectPortfolioBias).ToList());
```

Comment:
Could we directly apply the `.Where(RespectPortfolioBias)` when creating `lastActiveInsights` instead?